concepts/Common/Licensing/10 Validate Your License Key/15 Automated License Key File Creation.md

```diff
@@ -0,0 +1,23 @@
+You may need to use your license key in multiple places (your projects and CI systems). In such cases, you can store your DevExtreme license in key in an environment variable. You can then read that variable's value and create necessary files on demand.  
```

Comment:
(minor)
`In such cases, you can store your DevExtreme license in key`
maybe just 
`In such cases, you can store your DevExtreme license key`